app/src/main/java/com/globant/example/mentorapp/data/remote/APIService.java

```diff
@@ -0,0 +1,18 @@
+package com.globant.example.mentorapp.data.remote;
+
+import com.globant.example.mentorapp.data.entity.UserEntity;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+
+/**
+ * Created by oscar.amezquita on 7/06/2017.
+ */
+
+public interface APIService {
+
+    @GET("/repos/googlesamples/android-architecture/subscribers")
```

Comment:
I know this is the API you have to implement, but you're limiting your app to only work with this repo.

Features must be developed to be configurable and a good approach of this could be using parameters for login and repo name, so in that case you're providing a scalable functionality.